src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -0,0 +1,745 @@
+tags:
+    - NerdGraph
+    - Synthetic
+metaDescription: Use NerdGraph to programatically manage your synthetic monitors, private locations, and secure credentials.
+---
+
+Our NerdGraph API allows you to automate creating, updating, or deleting your synthetic monitors.
+
+If you've never used NerdGraph, read [Intro to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) for a more thorough tutorial. If you want to use the UI instead, check out [Add and edit monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors/).
+
+## Use queries to retrieve data [#query]
+
+Queries make requests to fetch data. Check out our [NerdGraph entities API tutorial](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial/) to learn additional query capabilities available to your synthetic monitors. 
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="query-examples"
+    title="Query monitors"
+  >
+
+  You query your synthetic monitors to get current status information about your data or details about the monitor configuration. Some query examples include: 
+
+```
+{
+  actor {
+    entitySearch(query: "domain = 'SYNTH' AND type = 'MONITOR'") {
+      results {
+        entities {
+          ... on SyntheticMonitorEntityOutline {
+            guid
+            name
+            accountId
+            monitorType
+            tags {
+              key
+              values
+            }
+          }
+        }
+      }
+    }
+  }
+}
+``` 
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="private-examples"
+    title="Query private locations"
+  >
+
+  You're able to similarly query your private locations:
+
+```
+{
+  actor {
+    entitySearch(query: "domain = 'SYNTH' AND type = 'PRIVATE_LOCATION'") {
+      results {
+        entities {
+          accountId
+          guid
+          name
+          tags {
+            key
+            values
+          }
+        }
+      }
+    }
+  }
+}
+``` 
+
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="credentials-example"
+    title="Query secure credentials
+"
+  >
+
+  Querying secure credentials lets you retrieve metadata only, but not the credential values values themselves:
+
+```
+{
+  actor {
+    entitySearch(query: "domain = 'SYNTH' AND type = 'SECURE_CRED'") {
+      results {
+        entities {
+          ... on SecureCredentialEntityOutline {
+            accountId
+            guid
+            name
+            tags {
+              key
+              values
+            }
+            updatedAt
+          }
+        }
+      }
+    }
+  }
+}
+```
+  </Collapser>
+</CollapserGroup>
+
+## Create your synthetic monitors 
+
+Below are some sample scripts to automate the creation of your synthetic monitors. Unlike with REST APIs, NerdGraph lets you programatically create [broken links](#create-broken-links), [step](#create-step), and [certificate check](create-certificate-check) monitors. 
```

Comment:
Id take out the reference to the REST APIs, in keeping with that talk we had about speaking in most docs as if those previous methods (REST APIs) no longer exist and just focusing on the new (and keeping transitional type stuff in a transitional guide of whatever sort)